openedge.properties

```diff
@@ -2,12 +2,18 @@ category=Languages
 description=Code Analyzer for OpenEdge
 homepageUrl=https://github.com/Riverside-Software/sonar-openedge
 archivedVersions=
-publicVersions=2.13.1
+publicVersions=2.13.1,2.14.4
 defaults.mavenGroupId=eu.rssw.sonar.openedge
 defaults.mavenArtifactId=sonar-openedge-plugin
 
+2.14.4.description=New expression engine in ABL Parser
+2.14.4.sqVersions=[7.9,9.1]
+2.14.4.date=2021-10-04
+2.14.4.changelogUrl=https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.14.4
+2.14.4.downloadUrl=https://github.com/Riverside-Software/sonar-openedge/releases/download/V2.14.4/sonar-openedge-plugin-2.14.4.jar
+
 2.13.1.description=Initial Marketplace entry. Support for OpenEdge 12.3 and 12.4
-2.13.1.sqVersions=[7.6,8.2]
+2.13.1.sqVersions=[7.6,8.9]
```

Comment:
Why are you suddenly upgrading the end of this range? There's already a replacement version out: 2.14.4. Further, the versions you're adding compatibility for are past EOL. The range on this old version needs to stay where it is, so users of the current LTS will adopt your latest version, rather than this old one.

Reply:
I was an error on my side, I thought those numbers reflected the minimum version of each major release, i.e. 7.6 and greater for 7.x branch, and 8.2 and greater for 8.x branch.
But as 2.13.1 is now an old version, this is not really important anymore